client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -164,20 +159,40 @@
     @Action hideAllVariables;
     @Action showAllVariables;
 
+    calcPolygon (variable: HMI.SimulationVariable): void {
```

Comment:
I understand that for d3 this data is a Polygon, but for us this is a band of values.
Maybe I'm picky but it would be clearer if the function and variables were named after what it represent.
Unless you thought of another situation were we can use polygons?

Reply:
I would prefer to keep the word `polygon` there in some way because it is important in understanding why the data structure is structured the way it is. I see your point though. What do you think of using the fn name `calcBandPolygon`?